docs/tracing/docs/030-details-troubleshooting.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Troubleshooting
+type: details
+---
+Using tracing you can troubleshoot different problems you might encounter while using Kyma. Understanding some common scenarios and how the expected traces look like will you have a better grasp on how to quickly use Kyma tracing capabilities to pin point the root cause.
```

Comment:
Tracing allows you to troubleshoot different problems that you might encounter while using Kyma. Understanding common scenarios and how the expected traces look gives you a better grasp on how to quickly use Kyma tracing capabilities to pinpoint the root cause.